HW5.py

```diff
@@ -0,0 +1,20 @@
+import csv, json
+parser = argparse.ArgumentParser()
+parser.add_argument('-csv')
+parser.add_argument('-json')
+ns = parser.parse_args(sys.argv[1:])
+
+csvFilePath = "user_details.csv"
+jsonFilePath = "user_details.json"
```

Comment:
These variables should be removed. Instead of them, you should use **ns.csv** and **ns.json**